bindings/python/tests/test_database.py

```diff
@@ -0,0 +1,90 @@
+import limbo
+import sqlite3
+
+
+def test_limbo_fetchall_select_all_users():
+    conn = limbo.connect("tests/database.db")
+    cursor = conn.cursor()
+    cursor.execute("SELECT * FROM users")
+
+    users = cursor.fetchall()
+    assert users
+    assert users == [(1, "alice"), (2, "bob")]
+
+
+def test_sqlite3_fetchall_select_all_users():
+    conn = sqlite3.connect("tests/database.db")
```

Comment:
No need to change this now, but you can use @pytest annotation to parametrize a _same_ test case to run with both libraries. See, for example, https://github.com/tursodatabase/libsql-experimental-python/blob/main/tests/test_suite.py

Reply:
Fixed